src/lib.rs

```diff
@@ -572,6 +546,33 @@ impl TestKit {
         self.poll_events();
     }
 
+    // Try to modify the test network configuration.
+    fn do_config_update(&mut self) {
+        let height = self.height();
+        if let Some(cfg_proposal) = self.cfg_proposal.take() {
+            match cfg_proposal {
+                ConfigurationProposalState::Uncommited(cfg_proposal) => {
```

Comment:
Nitpick: maybe, `use ConfigurationProposalState::*` within the method?

Reply:
Yes.